sdk/search/Azure.Search/tests/SessionRecords/SearchTests/PagingWithoutSize.json

```diff
@@ -0,0 +1,2481 @@
+{
```

Comment:
These are some large JSON files. Do we generally check these into the repo (especially PagingStaticDocumentsAsync.json/PagingStaticDocuments.json/etc.)?

As an aside:
Are the contents of the async/non-async json files identical?

Reply:
The json files are test recordings and yes, we typically check them into the repo.

Reply:
And the async/non-async are often identical but not always so.